src/main/java/bio/terra/app/controller/ApiValidationExceptionHandler.java

```diff
@@ -20,6 +22,25 @@
 @ControllerAdvice
 public class ApiValidationExceptionHandler extends ResponseEntityExceptionHandler {
 
+    /**
+     * Since the default {@link ResponseEntityExceptionHandler#handleExceptionInternal}
+     * method is often passed a null body, we override it and substitute the exception's error message
+     * into an ErrorModel and return that in the response instead.
+     */
+    @Override
+    protected ResponseEntity<Object> handleExceptionInternal(
+        Exception ex, @Nullable Object body, HttpHeaders headers, HttpStatus status, WebRequest request) {
+
+        if (HttpStatus.INTERNAL_SERVER_ERROR.equals(status)) {
+            request.setAttribute(WebUtils.ERROR_EXCEPTION_ATTRIBUTE, ex, WebRequest.SCOPE_REQUEST);
+        }
+
+        Object responseBody = body == null ? new ErrorModel().message(status.name() + " - see error details")
+            .addErrorDetailItem(ex.getMessage()) : body;
+
+        return new ResponseEntity<>(responseBody, headers, status);
```

Comment:
Not a huge deal, but IMO this is a case where overwriting a local is clearer than a ternary op
```suggestion
        if (body == null) {
            body = new ErrorModel().message(status.name() + " - see error details")         
                          .addErrorDetailItem(ex.getMessage());
        }
        return new ResponseEntity<>(body, headers, status);
```

Reply:
I'm not a fan of modifying input parameters, but I can get rid of the ternary operator.